scripts/check-ci-env.xsh

```diff
@@ -52,9 +55,10 @@ def _goto_error():
 
 _missing_env_var_settings = False
 _missing_env_var_secrets = False
+_missing_env_var_file_path = False
 
 # check if we are running in a GitLab CI or GitHub Actions environment
-if "GITLAB_CI" in os.environ or "CI" in os.environ:
+if "GITLAB_CI" not in os.environ or "CI" in os.environ:
```

Comment:
I do not get it, why this does not work for `GITLAB_CI` but work for `CI`, should not be `not or not` in the case?